tajo-common/src/test/java/org/apache/tajo/datum/TestArithmeticOperator.java

```diff
@@ -40,7 +40,7 @@ public TestArithmeticOperator(String option) {
     this.option = option;
   }
 
-  @Parameters
```

Comment:
Ok. Could you add this requirements to this page?
https://cwiki.apache.org/confluence/display/TAJO/Unit+Tests